src/commands/data/schemas/util/metaschema.json

```diff
@@ -232,24 +233,214 @@
+          "type": "string",
+          "enum": ["isStaffOfTargetPatient"]
+        }
+      },
+      "required": [
+        "type"
+      ]
+    },
+    "initiatorHasRelationToGroupInData": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "initiatorHasRelationToGroupInData"
+          ]
+        },
+        "groupIdField": {
+          "type": "string"
+        },
+        "relation": {
+          "type": "string",
+          "enum": ["staff", "patient"]
+        },
+        "requiredPermission": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "type"
+      ]
+    },
+    "objectConfiguration": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["object"]
+        },
+        "required": {
+          "type": "array",
+          "items": {
             "type": "string"
           }
         },
-        "required": [
-          "type"
-        ]
+        "queryable": {
+          "type": "boolean"
+        }
+      },
+      "additionalProperties": true,
+      "required": ["type"]
+    },
+    "arrayConfiguration": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["array"]
+        },
+        "items": {
+          "type": "object"
+        },
+        "minItems": {
+          "type": "number"
+        },
+        "maxItems": {
+          "type": "number"
+        },
+        "contains": {
+          "type": "object"
+        },
+        "queryable": {
+          "type": "boolean"
+        }
+      }
+    },
+    "stringConfigurations": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["string"]
+        },
+        "minLength": {
+          "type": "number"
+        },
+        "maxLength": {
+          "type": "number"
+        },
+        "enum": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "pattern": {
+          "type": "string"
+        },
+        "required": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "format": {
+          "type": "string",
+          "enum": ["date-time"]
+        },
+        "queryable": {
+          "type": "boolean"
+        }
+      }
+    },
+    "numberConfigurations": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["string"]
+        },
+        "minimum": {
+          "type": "number"
+        },
+        "maximum": {
+          "type": "number"
+        },
+        "enum": {
+          "type": "array",
+          "items": {
+            "type": "number"
+          }
+        },
+        "queryable": {
+          "type": "boolean"
+        }
       }
+    },
+    "booleanConfigurations": {
```

Comment:
I think you're missing `enum` and `const` here.

It's a little hidden, but some documentation can be found in this section of the documentation:
https://docs.extrahorizon.com/extrahorizon/services/manage-data/data-service/schemas#properties
![image](https://github.com/ExtraHorizon/exh-cli/assets/63499119/d696a352-564b-4175-a43a-a5964ccabcf6)

And I know that is actually missing the `queryable` property 😬

Reply:
Both const and enum are not in the OpenAPI specification.
I will add both `const` and `enum` to the schema. I have found a few other inconsistencies in the openAPI spec which I will document on confluence later this week

Reply:
Then we should create issues for updating the OpenAPI spec for the service as well indeed 😬 But good point to collect them all first 👍

Reply:
Jira ticket: https://extrahorizon.atlassian.net/browse/SVCS-1988